qa/common/shm_util.py

```diff
@@ -52,15 +53,26 @@ def create_set_shm_regions(input0_list, input1_list, output0_byte_size,
                            output1_byte_size, outputs, shm_region_names,
                            precreated_shm_regions, use_system_shared_memory,
                            use_cuda_shared_memory):
+    from infer_util import get_number_of_bytes_for_npobject
+
     if use_system_shared_memory and use_cuda_shared_memory:
         raise ValueError(
             "Cannot set both System and CUDA shared memory flags to 1")
 
     if not (use_system_shared_memory or use_cuda_shared_memory):
         return [], []
 
-    input0_byte_size = sum([i0.nbytes for i0 in input0_list])
-    input1_byte_size = sum([i1.nbytes for i1 in input1_list])
+    if input0_list[0].dtype == np.object:
```

Comment:
should this be `np.object` or `np.object_`?

Reply:
I think you have commented on the previous commit. It is fixed in the newer commit.